pkg/secrets-store/nodeserver.go

```diff
@@ -141,7 +141,7 @@ func (ns *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 		return &csi.NodePublishVolumeResponse{}, nil
 	}
 
-	klog.V(2).InfoS("node publish volume", "target", targetPath, "volumeId", volumeID, "attributes", attrib, "mount flags", mountFlags)
+	klog.V(2).InfoS("node publish volume", "target", targetPath, "volumeId", volumeID, "mount flags", mountFlags)
```

Comment:
consider printing the attributes but without the credential data .
e.g : 
- clone the attrib map ( callit attribNoCreds) 
- delete the CSIPodServiceAccountTokens  field from the clone map
- klogV2 with attribNoCreds map

Reply:
We already log it in after sanitizing the request here: https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/22b661d0bbaa896127726dbf9fd1c9c1e3d27230/pkg/secrets-store/server.go#L151, so doing the same sanitization again and logging is redundant.